CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Changes
+
+- Upgrade Facebook Login 12.2 => 16.3
```

Comment:
Perhaps we should mention the fact it is BC due to the manifest change and also inform that on our documentation (=> ticket).